Integrations/Flashpoint/Flashpoint.yml

```diff
@@ -0,0 +1,413 @@
+commonfields:
+  id: Flashpoint
+  version: -1
+name: Flashpoint
+display: Flashpoint
+category: Data Enrichment & Threat Intelligence
+description: "Use flashpoint integration for reduce business risk. \nThe integration\
+  \ was integrated and tested with demisto 5.0"
+configuration:
+- display: URL
+  name: url
+  defaultvalue: "https://fp.tools"
+  type: 0
+  required: true
+- display: API Key
+  name: api_key
```

Comment:
you are missing 2 common parameters.
```
- display: Trust any certificate (not secure)
  name: insecure
  required: false
  type: 8
- display: Use system proxy settings
  name: proxy
  required: false
  type: 8
```
also in the code use those two parameters.
take a look at this example: https://github.com/demisto/content/blob/master/Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

Reply:
Added both the params